module/lua/ldo.c

```diff
@@ -447,6 +459,11 @@ void luaD_call (lua_State *L, StkId func, int nResults, int allowyield) {
     else if (L->nCcalls >= (LUAI_MAXCCALLS + (LUAI_MAXCCALLS>>3)))
       luaD_throw(L, LUA_ERRERR);  /* error while handing stack error */
   }
+  intptr_t remaining = stack_remaining();
+  if (L->runerror == 0 && remaining < LUAI_MINCSTACK)
+    luaG_runerror(L, "C stack overflow");
+  if (L->runerror != 0 && remaining < LUAI_MINCSTACK / 2)
+    luaD_throw(L, LUA_ERRERR);  /* error while handing stack error */
```

Comment:
nit: s/handing/handling/

Reply:
LOL, this is copy-pasted from the comment above, I'll fix it too.